backend/typescript/middlewares/validators/authValidators.ts

```diff
@@ -65,9 +65,7 @@ export const registerRequestValidator = async (
         .status(400)
         .send(getApiValidationError("instagramLink", "string"));
     }
-    if (
-      req.body.businessName &&
-      !validatePrimitive(req.body.businessName, "string")
+    if (validatePrimitive(req.body.businessName, "string")
```

Comment:
I think this should be `!validatePrimitive(req.body.businessName, "string")`?
```suggestion
    if (!validatePrimitive(req.body.businessName, "string")
```

Reply:
oh no fixed!